modules/apps/object/object-test/src/testFunctional/tests/ObjectModelBuilder.testcase

```diff
@@ -184,4 +184,63 @@ definition {
 		}
 	}
 
+	@description = "LPS-185674 - Verify the user is able to view multiple object definitions in model builder view"
+	@priority = 4
+	test CanViewMultipleObjectDefinitions {
+		task ("Given several custom object definitions in a created folder") {
+			JSONObjectFolder.addFolder(
+				folderLabel = "Custom Folder 185674",
+				folderName = "CustomFolder185674");
+
+			ObjectAdmin.openObjectAdmin();
+
+			Click.javaScriptClick(
+				folderLabel = "Custom Folder 185674",
+				locator1 = "ObjectAdmin#SIDEBAR_ITEM");
+
+			for (var letter : list "A,B,C") {
+				ObjectAdmin.addNewObject(
+					fieldLabelName = "Custom Object 185674 ${letter}",
+					fieldName = "CustomObject185674${letter}",
+					pluralLabelName = "Custom Objects 185674 ${letter}");
+
+				ObjectAdmin.addObjectFieldViaAPI(
+					fieldBusinessType = "LongText",
+					fieldLabelName = "Custom Field",
+					fieldName = "customObjectField",
+					fieldType = "Clob",
+					isRequired = "false",
+					objectName = "CustomObject185674${letter}");
+
+				CreateObject.saveObject();
```

Comment:
You should save the object definition before creating fields by API. It'll return: `"HTTP 405 Method Not Allowed"` because the object definition never existed when `addObjectFieldViaAPI` is called.

Reply:
Another thing, we don't need to create object fields for this specifc test since we're only validating the object definitions on model builder view.